xwiki-rendering-api/src/main/java/org/xwiki/rendering/internal/transformation/DefaultTransformationManager.java

```diff
@@ -80,6 +80,9 @@
     private Provider<ComponentManager> componentManagerProvider;
 
     @Override
+    /**
+    * @deprecated
```

Comment:
This javadoc is not correct:

* the javadoc is supposed to be locate before all the annotations
* you should indicate that this javadoc inherits the on in `TransformationManager`, see http://dev.xwiki.org/xwiki/bin/view/Community/JavaCodeStyle#HDonotduplicateJavadoc. Any proper IDE should generate that for you (on Eclipse it's `Ctrl+Shift+j`)
* you should indicate what method should be used instead of this one